example-deployment/jwt-pub-key.pub

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
not sure if we need to include this ? should be generated, if i understand correctly.

Reply:
removed